website/client/components/chat/chatMessages.vue

```diff
@@ -268,6 +270,7 @@ import tier7 from 'assets/svg/tier-7.svg';
 import tier8 from 'assets/svg/tier-mod.svg';
 import tier9 from 'assets/svg/tier-staff.svg';
 import tier10 from 'assets/svg/tier-npc.svg';
+import bTooltip from 'bootstrap-vue/lib/directives/tooltip';
```

Comment:
hey just noticed this, with the new version of bootstrap-vue we're using it's not necessary to import directive and components manually, can you remove the import statement and this line https://github.com/HabitRPG/habitica/pull/9413/files#diff-e01bfe286bce1620d57ddaa25f228043R285?